src/Raven.Server/Documents/Includes/IncludeCountersCommand.cs

```diff
@@ -37,7 +37,24 @@ public IncludeCountersCommand(DocumentDatabase database, DocumentsOperationConte
         public IncludeCountersCommand(DocumentDatabase database, DocumentsOperationContext context, Dictionary<string, HashSet<string>> countersBySourcePath)
             : this(database, context)
         {
-            _countersBySourcePath = countersBySourcePath.ToDictionary(kvp => kvp.Key, kvp => kvp.Value.ToArray());
+            _countersBySourcePath = countersBySourcePath?.ToDictionary(kvp => kvp.Key, kvp => kvp.Value.ToArray())?? new Dictionary<string, string[]>();
```

Comment:
is there a place that countersBySourcePath can be null?

Reply:
fixed as we talked